files/dhcp/sethostname

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# This script is to update hostname of the system.
+
+if [ "$reason" != BOUND ] && [ "$reason" != RENEW ] \
+     && [ "$reason" != REBIND ] && [ "$reason" != REBOOT ]
+then
+    exit 0
+fi
+
+if [ "$interface" != "eth0" ] && [ "$interface" != "eth1" ]
```

Comment:
I feel this check is not needed since in our configuration we only do dhcp on eth0. We are not doing dhcp on other interfaces. If we decided to do dhcp on other interfaces, we'll explicit define that in the /etc/network/interfaces and won't need to add check here.